lib/Workflow/Manager.php

```diff
@@ -347,17 +321,17 @@ public function getTransitionByName(string $workflowName, string $transitionName
      * Forces an initial place being set (and stored) if the current place is empty.
      * We cannot apply a regular transition b/c it would be considered invalid by the state machine.
      *
-     * As of Symfony 4.4.8 built-in implementations of @see \Symfony\Component\Workflow\MarkingStore\MarkingStoreInterface
-     * use strict `null` comparison when retrieving the current marking and throw an exception otherwise.
-     *
-     * @param string $workflowName
+     * As of Symfony 4.4.8 built-in implementations of @param string $workflowName
      * @param object $subject
      *
      * @return bool true if initial state was applied
      *
      * @throws \Exception
+     *@see \Symfony\Component\Workflow\MarkingStore\MarkingStoreInterface
```

Comment:
comment moved to other place?

Reply:
There went something wrong while updating the php doc annotations. I restored the comment.